navigation/Point_xy start_pos
navigation/Point_xy goal_pos
sensor_msgs/LaserScan scan_list
---
bool ack
